Project Plan

```diff
@@ -0,0 +1,14 @@
+Topic - Correlation between Oil prices, currency valuation and consumer prices
+
+Questions - How does the cost of oil cause fluctuations in the valuation of various currencies and how does it affect the cost of items for consumers.
```

Comment:
I would take into account whether the country produces gas an oil as well

Reply:
In the notebook, as your building the project's story, start elaborating on the significance of this project. and what's the expected outcomes. 

what are we trying to understand from this analysis